Modelica/Clocked/ClockSignals.mo

```diff
@@ -0,0 +1,1227 @@
+    extends Modelica.Icons.SourcesPackage;
+
+    block PeriodicRealClock
+      "Generates a periodic clock signal with a period defined by a Real number"
+      parameter Modelica.SIunits.Time period
+        "Period of clock (defined as Real number)" annotation(Evaluate=true);
+      extends Clocked.ClockSignals.Interfaces.PartialPeriodicClock;
+    equation
+      if useSolver then
+         y = Clock(Clock(period), solverMethod=solverMethod);
+      else
+         y = Clock(period);
+      end if;
+
+      annotation (
+         defaultComponentName="periodicClock1",
+        Icon(coordinateSystem(preserveAspectRatio=true,  extent={{-100,-100},{100,100}}, initialScale=0.06),
+                        graphics={
+            Text(
+              extent={{-200,-110},{200,-175}},
+              textString="%period s"),
+            Rectangle(
+              extent={{20,58},{40,46}},
+              fillPattern=FillPattern.Solid,
+              rotation=45,
+              origin={-50,-120},
+              lineColor={95,95,95},
+              fillColor={95,95,95}),
+            Rectangle(
+              extent={{20,58},{40,46}},
+              fillPattern=FillPattern.Solid,
+              rotation=90,
+              origin={52,60},
+              pattern=LinePattern.None,
+              fillColor={95,95,95}),
+            Text(
+              visible=useSolver,
+              extent={{-300,-180},{300,-245}},
+              textString="%solverMethod")}),
+        Diagram,
+        Documentation(info="<html>
+<p>
+This component generates a periodic clock that starts ticking when
+the simulation starts. The <b>period</b> is defined in seconds
+by a <b>Real</b> number. If clocks are synchronized relatively to each other
+then only one of the clocks can be defined with PeriodicRealClock.
+</p>
+
+<p>
+For an introduction to clocks see
+<a href=\"modelica://Modelica.Clocked.UsersGuide.Clocks\">UsersGuide.Clocks</a>.
+If exact, integer based, time synchronization with absolute period definitions is
+desired, use block
+<a href=\"modelica://Modelica.Clocked.ClockSignals.Clocks.PeriodicExactClock\">PeriodicExactClock</a>
+to generate a periodic clock signal.
+</p>
+
+
+<p>
+If a clock is associated to a clocked continuous-time partition, then an <b>integrator</b>
+has to be defined that is used to integrate the partition from the previous
+to the current clock tick. This is performed by setting parameter <b>useSolver</b>
+= <b>true</b> and defining the integration method as String with
+parameter <b>solver</b>. Both parameters are in tab <b>Advanced</b>.
+For an example, see
+<a href=\"modelica://Modelica.Clocked.Examples.Systems.ControlledMixingUnit\">Examples.Systems.ControlledMixingUnit</a>.
+</p>
+</html>"));
+    end PeriodicRealClock;
+
+    block PeriodicExactClock
+      "Generates a periodic clock signal with a period defined by an Integer number with resolution"
+      import R = Modelica.Clocked.Types.Resolution;
```

Comment:
The renaming (and shortening) import is no longer endorsed (even though it is used in the specification a lot). Better use `import Modelica.Clocked.Types.Resolution;`.